### ISSUE TITLE
Sidebar settings error

### ISSUE THREAD

**Issue by GitTheHellOutaHere** [iina/iina#2048]
There’s an error in the sidebar Video pane.
It says "Equalizer" above the picture settings.

I imagine this has been corrected in one of the beta versions, but just in case...

![iina error](https://user-images.githubusercontent.com/15519219/47611984-eb3fa380-da2d-11e8-9b90-9367b865d1df.png)

v0.0.15.1

**Comment by lhc70000** [comment]
This is what mpv call these options: https://mpv.io/manual/master/#equalizer.

**Comment by vbwx** [comment]
I wanted to report the same thing. I thought it was a simple localization error, as the equalizer settings in the audio panel are called "Video Adjustment".

<img width="356" alt="Audio Panel" src="https://github.com/iina/iina/assets/10358990/4b95ac8d-8931-4941-9f34-0b1a69889bb9">

Using v1.3.3.

**Comment by svobs** [comment]
ATTN @low-batt: @vbwx's observation is legit - take a look at `QuickSettingViewController.strings` (en-GB), objectID `iS4-Zr-9Ig`.

```
/* Class = "NSTextFieldCell"; title = "Equalizer"; ObjectID = "iS4-Zr-9Ig"; */
"iS4-Zr-9Ig.title" = "Video Adjustment";
```

Should be "Equalizer".

**Comment by GitTheHellOutaHere** [comment]
I gave up on iina long ago - mostly use [Elmedia Player](https://www.elmedia-video-player.com/) now. Dumb name, but no worse than iina. It often plays files VLC won't - especially partially-downloaded ones.

VLC is barely maintained anymore. A [thread](https://forum.videolan.org/viewtopic.php?f=7&t=84522) was started in Nov. '18 that requested a fix for an annoying interface issue. Since then, many others have complained and requested a fix as well. Although this 'bug' has affected and angered many users for years, it's been ignored in about 55 subsequent [updates.](https://en.wikipedia.org/wiki/VLC_media_player#Release-history)

In July '23, one VLC developer responded to the thread: **"I am not sure if you realise that VLC is a volunteer project. Other people don't necessarily have the time, skill or motivation to fix what _you_ care about, or even much time at all to dedicate to the project.**

In other words, if you don't like it, don't use it. Don't ask for, or expect, software support - regardless of whether or not you donated to the project.

**Comment by low-batt** [comment]
Hi @GitTheHellOutaHere,

This junior IINA developer very much appreciates all of the IINA feedback posted by GitHub users including harsh critical comments.

IINA relies upon volunteers, software engineers overloaded with work/school that somehow find time to work on IINA. Lately development has been steadily moving ahead, a bit at glacial speed.

On the issue at hand…

The original problem reported was that the section of the quick settings panel `Video` tab that contains the settings `Brightness`, `Contrast`, etc. is titled `Equalizer`. I too wondered about that but if you search the net you will fine the term "video equalizer" being used for this. IINA is a [GUI frontend](https://github.com/mpv-player/mpv/wiki/Applications-using-mpv#gui-frontends) for [mpv](https://mpv.io/) so it is desirable for IINA to use `mpv` terminology. As @lhc70000 mentioned above the `mpv` options these sliders control are documented in the [Equalizer](https://mpv.io/manual/stable/#equalizer) section of the [mpv manual](https://mpv.io/manual/stable/). I will check with other developers, but this seems like an acceptable name for this section of the quick settings panel.

On the recent report from @vbwx that the similar section on the `Audio` tab is titled `Video Adjustment` when language is configured to `en-GB`, that is a clear defect. I believe that needs to be fixed in [Crowdin](https://translate.iina.io/).

As we are currently fixing localization problems I have labeled this issue as high priority to fix in the next release.

@GitTheHellOutaHere Thank you for raising this issue.

**Comment by uiryuu** [comment]
The Audio Equalizer doesn't displayed as "Video Adjustment" anymore for en-GB in the latest develop branch.


![Image](https://github.com/iina/iina/assets/20237141/04aa2042-3653-4b97-adf2-ecc05be8ac45)
